spec/scrum_lint/models/card_spec.rb

```diff
@@ -1,6 +1,7 @@
 RSpec.describe ScrumLint::Card do
   let(:card_params) do
     {
+      board_name: nil,
```

Comment:
Required param for instantiating a [`ScrumLint::Card`](https://github.com/thread-pond/scrum-lint/blob/master/lib/scrum_lint/models/card.rb#L12)